jax/_src/numpy/lax_numpy.py

```diff
@@ -2521,39 +2521,62 @@ def _pad(array, pad_width, mode, constant_values, stat_length, end_values, refle
   if mode == "constant":
     return _pad_constant(array, pad_width, constant_values)
 
-  elif mode == "wrap":
```

Comment:
I have a slight preference for the sequence of `elif` because it more clearly expresses a case statement, without having to read all the branches to ensure that there is no fall-through. WDYT?

Reply:
Thanks for the review:)
I agree with your idea considering it is a fairly long case statement.
Learn a new thing!